test/compilable/test16607.d

```diff
@@ -0,0 +1,15 @@
+struct A(T)
+{
+    T t; // causes A to be SIZEOKfwd b/c B (passed as T) isn't yet done
```

Comment:
Mmh, only unwillingly, I'm fairly used to that style and have a lot of stuff to type each day.
I wouldn't do that in a changelog entry or specs, but I don't see how that increases any hurdle.
If you're already starring at test16607.d, you're deep into mud anyhow.